_posts/2019-01-30-renting-a-property-continuing-our-discovery.md

```diff
@@ -0,0 +1,77 @@
+- Who is it a problem for?
+- When is it happening?
+
+Without knowing these there could be a missed opportunity to meet the right set of needs. There is an increased risk of failure to create impact without the who, what and when being evidenced prior to ideation of concepts.
+
+Through the research in phase one it was clear there were complex relationship dynamics associated with renting. Not only were they person to person they were also between people and property. In order to better understand the impact of relationships it was crucial to define how each one worked and what good looked like. A good relationship can include the following:
+
+- Issues are raised without fear
+- Issues are resolved in a timely manner
+- Rapport - (know names, situation etc)
+- Consideration of actions
+- Going above and beyond requirements
+- Taking pride in their house and responsibilities
+- Autonomy
+- Maintain the mana and integrity of the other person or persons in the relationship
+
+We noted that defined responsibilities and expectations are important to reducing conflict or managing it when it arises through accountability. Similarly, transparency encourages fairness and equality as well as manages expectations of both parties.
+
+## The outcomes
+
+Through the analysis and validation we identified these key insights:
+
+1. There is strong evidence that the barriers to a ‘quality relationship’ between the parties to a tenancy start before the actual tenancy commences.
+2. There is an opportunity to be pre-emptive as it is the time before a tenancy starts in which the relationship between landlords and tenant is most critical. The events of this time have a long term impact throughout renting life journey.
+3. There is strong evidence that removing known barriers at the beginning of the renting relationship can reduce the likelihood of issues and conflict arising during a tenancy and increases a quick resolution.  
+
+Based on these insights two key issues stood out that are occurring before people start renting a property.
+
+- Securing a property to rent
+- Forming a tenancy agreement.
+
+We identified segments of people that are struggling to acquire a rental property. This could be for a variety of reasons but we think would be an ideal cohort to work with as there is a good opportunity to achieve collective impact by applying a cross agency approach with these groups.
+
+## What informed these outcomes
+
+- Defining and mapping the 31 unique tenancy relationships
+- Barriers to good relationships definitions and mapping
+- Landlord and Tenant Audiences - Segmentation, Archetypes, Personas
+- Landlord and Tenant journey and barriers maps
+- Quantitative data analysis of discovery phase relationship findings
+- Logic mapping of the seven concepts from Phase One for validation against project goals
+- Māori concept of [Kāinga](https://www.digital.govt.nz/blog/kainga-and-renting-a-property/)
+
+## What was the research used for? 
```

Comment:
you've got trailing whitespace here. Also the style guide says no questions in titles.